WordPress/Classes/ViewRelated/Blog/Blog Dashboard/DashboardCard.swift

```diff
@@ -65,19 +65,19 @@ enum DashboardCard: String, CaseIterable {
         }
     }
 
-    private func shouldShowRemoteCard(apiResponse: BlogDashboardRemoteEntity?) -> Bool {
-        guard let apiResponse = apiResponse else {
+    private func shouldShowRemoteCard(postsInfo: BlogDashboardPostsInfo?) -> Bool {
```

Comment:
I think this is too specific to posts, what do you think about returning the `apiResponse` here and having the `BlogDashboardPostsInfo` inside it?

The reason is because I can already see a case in which this will need to be changed:

1. Take into account the currently in-development Blogging Prompts card
2. We don't want to show this card and the "nextPost" at the same time
3. Here we would check (from the API) if the blogging prompts card should appear
4. However... all we have is the posts info.

Wdyt?

Reply:
I completely agree with you!
The posts info approach was actually problematic for other reasons as well. I completely removed it while fixing an issue here 141b43042e3425b9eabecb292f764b39aabe76e8